goverance.md

```diff
@@ -0,0 +1,67 @@
+
+## Working Group Roles
+
+The SAFE Working group aims to have three Chairs, Technical Leads for each substantive block of work, and many members.
+
+Unless otherwise stated, individuals are expected to be responsive and active within
+their roles.  Within this section "member" refers to a member of a Chair, Technical Lead or
+Members.
+
+### Role of Members
+- The primary role of a member is to contribute expertise to the group.
+- Initial members are defined at the founding of the WG.
+- To add yourself as a member, submit a PR adding yourself to the list of members.
+- Members *SHOULD* remain active and responsive in their Roles.
+- Members taking an extended leave of 1 or more months *SHOULD* coordinate with other members to ensure the role is adequately staffed during the leave.
+- Members going on leave for 1-3 months *MAY* work with other members to identify a temporary replacement.
+- Members of a role *SHOULD* remove any other members that have not communicated a leave of absence and either cannot be reached for more than 1 month or are not fulfilling their documented responsibilities for more than 1 month. This may be done through a [super-majority] vote of members, or if there are not enough *active* members to get a super-majority of votes cast, then removal may occur through a [super-majority] vote between Chairs and Technical Leads.
+- Membership disagreements may be escalated to the WG Chairs.  WG Chair membership disagreements may be escalated to the CNCF Steering Committee.
+- Members *MAY* decide to step down at anytime and propose a replacement.  
+- The group will use lazy consensus amongst other members with fallback on majority vote to accept proposal.  The candidate *SHOULD* be supported by a majority of SIG Members or Subproject Contributors (as applicable).
+- Members *MAY* select additional members through a [super-majority] vote amongst members. This *SHOULD* be supported by a majority of WG Members.
```

Comment:
What's the difference between suggesting members via super-majority vote vs the earlier bullet, `To add yourself as a member, submit a PR adding yourself to the list of members.`? Is the expected flow going forward that if you want to be a Member, you submit a PR and the group votes?

Reply:
The super-majority vote wasn't intended for members. Removing this.